server/channels/app/channel.go

```diff
@@ -1933,7 +1933,7 @@ func (a *App) GetChannelByName(c request.CTX, channelName, teamID string, includ
 	return channel, nil
 }
 
-func (a *App) GetChannelsByNames(c request.CTX, channelNames []string, teamID string) ([]*model.Channel, *model.AppError) {
+func (a *App) GetChannelsByNames(channelNames []string, teamID string) ([]*model.Channel, *model.AppError) {
```

Comment:
c was an unused parameter. Hence removed

Reply:
That sounds like it should be fine, but I'll double check with the team to see if there's something else we could pass from the email batching code as context instead

Reply:
Okay, we actually have an effort to pass context to every `Store` method so that those can be cancelled, so we shouldn't remove this from here. Could you re-add them and have the email batching code call this method by passing `request.EmptyContext` instead?

There's not much more info in the thread, but If you're on our Community server and you're curious, here's the link to it: https://community.mattermost.com/core/pl/y5obqr8r7p8sjk1obdizksx55o

Reply:
Done